recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
     sha256: dceb15fef556a2775121bcc695561df4ff0e09248e0ce3a2d58f5244a9f61421
 
 build:
-    number: 0
+    number: 1
```

Comment:
I'm not sure it is necessary to update the build number?

We will create updated builds for packages like: 
`pytables-3.3.0-np111py35_0.tar.bz2` to 
`pytables-3.3.0-np111py35_1.tar.bz2`

I wonder if this is going to cause unnecessary upgrades.